karma.conf.js

```diff
@@ -1,3 +1,33 @@
+const fs = require('fs');
+
+// Use ENV vars on Travis and sauce.json locally to get credentials
+if (!process.env.TRAVIS && !process.env.SAUCE_USERNAME) {
+  if (!fs.existsSync('./sauce.json')) {
+    console.log('Please, create a sauce.json with your SauceLabs credentials.');
+    process.exit(1);
```

Comment:
This means that it is required to have Saucelabs account in order to run the tests even locally.

Maybe we should just print a warning and go ahead and use Chrome as it is configured further down.